cpp/test/Slice/errorDetection/EnumeratorValues.err

```diff
@@ -1,5 +1,5 @@
 EnumeratorValues.ice:9: value for enumerator 'C' is out of range
```

Comment:
Now instead of just saying "out of range", we say what the value was, and specifically state why it's bad, so the user knows what the "allowed range" is from the message.